pkg/cache/internal/cache_reader.go

```diff
@@ -253,3 +277,9 @@ func KeyToNamespacedKey(ns string, baseKey string) string {
 	}
 	return allNamespacesNamespace + "/" + baseKey
 }
+
+// KeyToClusteredKey prefixes the given index key with a cluster name
+// for use in field selector indexes.
+func KeyToClusteredKey(clusterName string, ns string, baseKey string) string {
+	return clusterName + "/" + KeyToNamespacedKey(ns, baseKey)
```

Comment:
Internally we use `cluster|namespace/name`. changing here?

Reply:
yep, let's use the chance

Reply:
done